packages/vscode-extension/src/handlers.ts

```diff
@@ -351,6 +351,7 @@ export async function createNewProjectHandler(args?: any[]): Promise<Result<any,
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.CreateProjectStart, getTriggerFromProperty(args));
   const result = await runCommand(Stage.create);
   if (result.isOk()) {
+    await ExtTelemetry.dispose();
```

Comment:
Just curious that `ExtTelemetry.Reporter` has been added to vscode context's subscriptions, will vscode call dispose on reloading?

Reply:
Yes, in theory it will and it works for my testing. But currently we have[ telemetry issue](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12974459?src=WorkItemMention&src-action=artifact_link) that many telemetry events are not sent and this only happens for create project scenario. So we want to manually flush the telemetry reporter to make sure it is sent before vscode reload. This is just a potential fix and we also would like to see whether this will have effect on the issue.

Reply:
I see. So in your test will the reported be disposed twice, any error if dispose twice?

Reply:
Yes, vscode TelemetryReporter.dispose() is called twice and as far as I can see, that's no error occurred.